src/foundation/type/index.stories.js

```diff
@@ -73,6 +73,14 @@ storiesOf('foundation|Type', module)
           <p className='mc-text--muted mc-text--monospace'>.mc-text-intro</p>
         </div>
 
+        <div className='align-items-center example--section'>
+          <h5 className='mc-text-h5'>Readable</h5>
+          <p className='mc-text-intro'>Online classes taught by the world&#39;s greatest minds. Now get
```

Comment:
uses `mc-text-intro`

not technically a problem because they share the same styles, but that might not always be the case